<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox_PortName">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&lt;Обновить&gt;</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_PortSpeed">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>406000</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_Connect">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Соединить</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="consol">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>421</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_madr">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
